frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -48,6 +50,28 @@ public Model(ModelArchive modelArchive, int queueSize) {
                         this.modelArchive.getModelName(), this.modelArchive.getModelVersion());
     }
 
+    public JsonObject getModelState(boolean isDefaultVersion) {
+
+        JsonObject modelInfo = new JsonObject();
+        modelInfo.addProperty("defaultVersion", isDefaultVersion);
```

Comment:
In general, keep all these const string names to a global location.
Same here. const strings / prop names to either global location or use getter methods.